lib/msal-common/src/logger/Logger.ts

```diff
@@ -64,7 +64,7 @@ export class Logger {
         };
         this.localCallback = loggerOptions.loggerCallback || defaultLoggerCallback;
         this.piiLoggingEnabled = loggerOptions.piiLoggingEnabled || false;
-        this.level = loggerOptions.logLevel || LogLevel.Info;
+        this.level = typeof(loggerOptions.logLevel) === "number" ? loggerOptions.logLevel : LogLevel.Info;
```

Comment:
Do we want to verify it's a valid LogLevel and if not set it to LogLevel.Info as well

Reply:
That's what I implemented originally, but we agreed that the validation logic added too much complexity. Here's why we think this is the appropriate change:

1. This catches and handles the "non-number" issue
2. Any number higher than the max supported level will result in an equivalent to setting the level to `Trace`
3. Any number lower than the minimum supported (0 for Error) will result in an equivalent to setting the level to `Error`
4. We provide the `LogLevel` enum, so developers generally don't need to set the level numeric value themselves.